src/probability/martingale/basic.lean

```diff
@@ -553,6 +553,21 @@ lemma submartingale_iff_expected_stopped_value_mono [is_finite_measure μ]
 ⟨λ hf _ _ hτ hπ hle ⟨N, hN⟩, hf.expected_stopped_value_mono hτ hπ hle hN,
  submartingale_of_expected_stopped_value_mono hadp hint⟩
 
+lemma submartingale.stopped_process [is_finite_measure μ]
```

Comment:
Could you please add a docstring and make it `protected`? Otherwise LGTM. Thanks!
bors d+